### PR TITLE
Change Readme

### PR DESCRIPTION
**Pull request by Spidy123222**
This changes the readme to one meant for Sidestore. This is currently a draft since the project is still progressing and you guys can edit this if you feel it needs to be changed before merging.

### CODE REVIEW

**Comment by Simonx22** [comment]
Is this project called SideStore or Sidestore? The GitHub organization and repo is called “SideStore” but in the readme it’s called “Sidestore”.

**Comment by Spidy123222** [comment]
> I was just about to comment the same thing, I’ve pulled out my iPad to point out some other things.

yeah wasn't quite sure if it should so I used Sidestore.

**Comment by jkcoxson** [comment]
It should be upper camel case (SideStore)

**Comment by Spidy123222** [comment]
> It should be upper camel case (SideStore)

It has been fixed now.

File: README.md
Comment (jkcoxson) [inline README.md:46]: We can leave Roxas, just change it to mention that it's Riley's framework. We're still using it as a dependency.

File: README.md
Comment (jkcoxson) [inline README.md:28]: Maybe change to 'It is programmed in the Rust programming language and it open source'
And you can set Netmuxd to be a hyper link like [Netmuxd](https://github.com/jkcoxson/netmuxd)

File: README.md
Comment (jkcoxson) [inline README.md:66]: This is unnecessary because you said to do it in step 3

File: README.md
Comment (Spidy123222) [reply README.md:46]: solved on latest commit

File: README.md
Comment (jkcoxson) [inline README.md:79]: Maybe just state that it's licensed under AGPLv3. I don't like the tone of this.

File: README.md
Comment (Spidy123222) [reply README.md]: Changed in latest commit

File: README.md
Comment (Spidy123222) [reply README.md]: Changed in latest commit

File: README.md
Comment (Spidy123222) [reply README.md]: Changed in latest commit

File: README.md
Comment (Spidy123222) [reply README.md:46]: i have added roxas thanks for mentioning its Changed in latest commit

File: README.md
Comment (jkcoxson) [inline README.md:31]: Testut's, Alt**S**tore

File: README.md
Comment (jkcoxson) [inline README.md:47]: Misspelled SideStore

Comment (Spidy123222) [reply README.md:47]: fixed it in https://github.com/SideStore/SideStore/pull/11/commits/be79f1a1f5d29ac16bd4a0ce4bb0e9aac85afd63

Comment (Spidy123222) [reply README.md]: This should fix that https://github.com/SideStore/SideStore/pull/11/commits/40660e459e0d662eca5fb6ffcd284fa2e9338994

File: README.md
Comment (Spidy123222) [reply README.md]: fixed in https://github.com/SideStore/SideStore/pull/11/commits/40660e459e0d662eca5fb6ffcd284fa2e9338994

**Comment by jkcoxson** [comment]
I’ve already reviewed it smh, we’re waiting on 1 other approval

**Comment by Spidy123222** [comment]
> I’ve already reviewed it smh, we’re waiting on 1 other approval

Yeah sorry I miss clicked I didn’t know there was a way to request for a re-review at bottom of page sorry about that. I thought it was a way to refresh.

**Review by JoeMatt — APPROVED** [review]

**Pull request merged**
